Wed June 6 2007 Dongmin Zhang <crosstool@gmail.com>

Change the default number of connections per minute allowd for the same program by inetd.
This allows more connections per minute by inetd.

--- inetutils-1.4.2/inetd/inetd.c	2002-06-25 20:15:06.000000000 -0700
+++ inetutils-1.4.2.new/inetd/inetd.c	2007-05-29 17:05:09.000000000 -0700
@@ -127,7 +127,7 @@
 #endif
 #include <grp.h>
 
-#define	TOOMANY		40		/* don't start more than TOOMANY */
+#define	TOOMANY		1000		/* don't start more than TOOMANY */
 #define	CNT_INTVL	60		/* servers in CNT_INTVL sec. */
 #define	RETRYTIME	(60*10)		/* retry after bind or server fail */
 
